west.yml

```diff
@@ -73,7 +73,7 @@ manifest:
       groups:
         - hal
     - name: hal_gigadevice
-      revision: 749fdd4b9dc9dfc57daac1229a5a669109e14624
```

Comment:
nit: this commit should come first (otherwise bisect will be broken for a range of commits)

Reply:
done